templates/zerver/login.html

```diff
@@ -186,8 +186,7 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
         https://stackoverflow.com/questions/5283395/url-hash-is-persisting-between-redirects */
         var email_formaction = $("#login_form").attr('action');
         $("#login_form").attr('action', email_formaction + '/' + window.location.hash);
-        $("#google_login_form input[name='next']").attr('value', '/' + window.location.hash);
-        $("#social_login_form input[name='next']").attr('value', '/' + window.location.hash);
+        $(".social_login_form input[name='next']").attr('value', '/' + window.location.hash);
```

Comment:
This actually fixes a pre-existing bug with the AzureAD one, right?

Reply:
Yeah, I should have extracted it separately.